src/hotspot/share/prims/methodHandles.cpp

```diff
@@ -907,10 +907,10 @@ void MethodHandles::expand_MemberName(Handle mname, int suppress, TRAPS) {
       }
       InstanceKlass* defc = InstanceKlass::cast(java_lang_Class::as_Klass(clazz));
       DEBUG_ONLY(clazz = nullptr);  // safety
-      int vmindex  = java_lang_invoke_MemberName::vmindex(mname());
+      intptr_t vmindex  = java_lang_invoke_MemberName::vmindex(mname());
```

Comment:
Why not do the checked_cast<int> here instead of below?
Ideally, the vmindex field would be changed to int (field offsets, itable/vtable indexes are all int), but that's more work.

Reply:
> Why not? It's the same and casts the int where it's needed to be an int.
To me the checked_cast looks less ugly in the initialization vs at the uses sites, but in this case with only one use it doesn't really matter.